src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,9 +43,19 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        String[] keywordsArr = keywords.toArray(new String[0]);
+        for(int i = 0; i < keywordsArr.length; i++) {
+            keywordsArr[i] = keywordsArr[i].toLowerCase();
+        }
+        keywords = Set.of(keywordsArr);
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            String[] wordsInNameArr = wordsInName.toArray(new String[0]);
+            for(int i = 0; i < wordsInNameArr.length; i++) {
+                wordsInNameArr[i] = wordsInNameArr[i].toLowerCase();
+            }
+            wordsInName = Set.of(wordsInNameArr);
```

Comment:
Why not just put a List<String> of lowercase names into the new HashSet<>( ... )?
That way, it reduces the trouble of you converting it to array and then back to set again.